code/game/bioscans.dm

```diff
@@ -94,15 +94,20 @@ GLOBAL_DATUM_INIT(bioscan_data, /datum/bioscan_data, new)
 	var/marine_planet_location_string = "[marine_planet_location ? ", including one in [marine_planet_location]." : "."]"
 	var/marine_ship_location_string = "[marine_ship_location ? ", including one in [marine_ship_location]." : "."]"
 
+	var/ghost_scan = SPAN_ALERT("[xenos_on_planet] xenos on planet, with [larva] larva. [xenos_on_ship] xenos on the ship. [marines_on_planet] humans on the planet. [marines_on_ship] humans on the ship.")
```

Comment:
Was it intentional you left out the new lines for this one (conversely ares bioscan still have new lines in the log)? If so, everything looks good to go.

Reply:
Nope!